node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/file/Editor.java

```diff
@@ -102,7 +102,7 @@ public boolean edit(Consumer<String> logConsumer) {
             return false;
         }
 
-        String diffDescription = diffTooLarge(diff) ? "" : ":\n" + diff.toString();
+        String diffDescription = diffTooLarge(diff) ? ": Diff too large" : ":\n" + diff.toString();
```

Comment:
Idea: How about we try to track the size the diff would have had and print that here? Now we have no idea how much larger we need to raise MAX_LENGTH to be able to see the diffs.

Reply:
Sure, now logs 
```
NodeAgent-host002-10: Patching file ip6tables-filter: Diff too large (518)
```